addons/survey/models/survey_question.py

```diff
@@ -710,35 +724,60 @@ class SurveyQuestionAnswer(models.Model):
     sequence = fields.Integer('Label Sequence order', default=10)
     scoring_type = fields.Selection(related='question_id.scoring_type')
     # answer related fields
-    value = fields.Char('Suggested value', translate=True, required=True)
+    value = fields.Char('Suggested value', translate=True)
     value_image = fields.Image('Image', max_width=1024, max_height=1024)
     value_image_filename = fields.Char('Image Filename')
+    value_label = fields.Char('Value Label', compute='_compute_value_label', translate=True,
```

Comment:
Translate is not really relevant (we don't translated computed fields, it will be translated based on fields fetched for its computation)